pkg/syncer/cnsoperator/controller/cnsnodevmattachment/cnsnodevmattachment_controller.go

```diff
@@ -340,18 +342,35 @@ func (r *ReconcileCnsNodeVMAttachment) Reconcile(ctx context.Context, request re
 	if instance.DeletionTimestamp != nil {
 		nodeVM, err := dc.GetVirtualMachineByUUID(ctx, nodeUUID, false)
 		if err != nil {
```

Comment:
Do we get a specific error in case of not authenticated error?

Reply:
its types.NotAuthenticated. But with vc.Connect() now this should never happen.